lib/spans/span-event-aggregator.js

```diff
@@ -92,6 +92,9 @@ class SpanEventAggregator extends EventAggregator {
 
     this.periodMs = periodMs
     this.limit = limit
+    this._metrics
+      .getOrCreateMetric(`${NAMES.SUPPORTABILITY.PREFIX}SpanEvent/Limit`)
+      .recordValue(this.limit)
```

Comment:
Minor (and this is already merged) but we do have a collection of 'SpanEvent' support metrics organized in the 'names' file that has our various metric names/prefixes/etc.

https://github.com/newrelic/node-newrelic/blob/main/lib/metrics/names.js#L233-L237

For future similar changes would prob organize in there.